rotkehlchen/chain/ethereum/modules/compound/compound.py

```diff
@@ -1,233 +1,23 @@
 import logging
-from collections import defaultdict
-from typing import TYPE_CHECKING, Any, Optional
 
 from rotkehlchen.accounting.structures.balance import Balance
 from rotkehlchen.assets.asset import Asset
-from rotkehlchen.chain.ethereum.modules.compound.constants import CPT_COMPOUND
-from rotkehlchen.chain.ethereum.modules.compound.utils import CompoundBalance
-from rotkehlchen.chain.ethereum.modules.compound.v2.compound import CompoundV2
-from rotkehlchen.chain.ethereum.modules.compound.v3.compound import CompoundV3
-from rotkehlchen.constants import ZERO
-from rotkehlchen.db.filtering import EvmEventFilterQuery
-from rotkehlchen.db.history_events import DBHistoryEvents
-from rotkehlchen.history.events.structures.evm_event import EvmEvent
-from rotkehlchen.history.events.structures.types import HistoryEventSubType
-from rotkehlchen.history.price import query_usd_price_zero_if_error
-from rotkehlchen.inquirer import Inquirer
 from rotkehlchen.logging import RotkehlchenLogsAdapter
-from rotkehlchen.premium.premium import has_premium_check
-from rotkehlchen.types import ChecksumEvmAddress, Timestamp
+from rotkehlchen.types import ChecksumEvmAddress
 from rotkehlchen.utils.interfaces import EthereumModule
-from rotkehlchen.utils.misc import ts_ms_to_sec
-
-if TYPE_CHECKING:
-    from rotkehlchen.chain.ethereum.defi.structures import GIVEN_DEFI_BALANCES, GIVEN_ETH_BALANCES
-    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
-    from rotkehlchen.db.dbhandler import DBHandler
-    from rotkehlchen.premium.premium import Premium
-    from rotkehlchen.user_messages import MessagesAggregator
 
 ADDRESS_TO_ASSETS = dict[ChecksumEvmAddress, dict[Asset, Balance]]
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
-class Compound(EthereumModule, CompoundV2, CompoundV3):
+class Compound(EthereumModule):
```

Comment:
This module is now empty (?) Should we remove it from the settings in the db upgrade? So remove it completly